.github/workflows/ios-build.yml

```
@@ -7,30 +7,40 @@ on:
+      - 'ios/**'
       - '**/Podfile.lock'
+      - '**/bun.lock'
       - '**/*.podspec'
       - '**/react-native.config.js'
       - '**/nitro.json'
 
 env:
   USE_CCACHE: 1
 
+concurrency:
+    group: ${{ github.workflow }}-${{ github.ref }}
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)